LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/CommonAPI/LoRaFPort.cs

```diff
@@ -11,7 +11,7 @@ public static class LoRaFPort
         /// <summary>
         /// Starting Fport value reserved for future applications.
         /// </summary>
-        public const byte ReservedForFutureAplications = 224;
+        public const byte ReservedForFutureAplications = 225;
```

Comment:
According to [LoRaWAN specification](https://lora-alliance.org/wp-content/uploads/2020/11/lorawan1.0.3.pdf), "FPort values 225..255 (0xE1..0xFF) are reserved for future standardized application 
extensions."